### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by ndushay**
## Why was this change made? 🤔

In the vain hopes it will help us better recognize when to run integration tests.

## How was this change tested? 🤨

⚡ ⚠ If this change has cross service impact, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 ## How was this change tested? 🤨
 
-⚡ ⚠ If this change has cross service impact, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡
+⚡ ⚠ If this change has cross service impact, including writes to file system, ***run [integration tests](https://github.com/sul-dlss/infrastructure-integration-test)*** and/or test in [stage|qa] environment, in addition to specs. ⚡
```
Comment (jcoyne) [inline .github/pull_request_template.md:7]: I'm not sure this is specific enough.  Can we say a "shared file system"?

Comment (ndushay) [reply .github/pull_request_template.md:7]: good idea, but I'm not sure that folks will know from the code when we're writing to a shared file system.

Comment (ndushay) [reply .github/pull_request_template.md:7]: I understand this;  I think everyone on the team understands this.  Not everyone on the team will know what is being written to a "shared file system" or which ones we mean, or want to go looking through shared_configs and puppet to find out

Comment (aaron-collier) [reply .github/pull_request_template.md:7]: I think for clarity, @jcoyne's suggestion makes sense to me. Or perhaps something that indicates non-logging based writes like `including writes to file system (exceptions for simple logging, etc.)`.

Comment (mjgiarlo) [reply .github/pull_request_template.md:7]: Agree that @jcoyne's tweak makes sense in this case.

Comment (ndushay) [reply .github/pull_request_template.md]: ok, updated!   Would be delighted if someone merged

**Review by aaron-collier — APPROVED** [review]

**Pull request merged**